### ISSUE TITLE
Transfer learning performance of DINO on more challenging downstream dense prediction tasks.

### ISSUE THREAD

**Issue by Yuxin-CV** [facebookresearch/dino#127]
Thanks for your great work!

I believe a goal of un-/self-supervised learning is to learn transferrable feature representations. I notice that DINO conduct a study on some smaller image classification datasets such as CIFAR-10/-100, and the performance is quite impressive.

But it seems that the performance of modern neural nets on these image classification datasets is somewhat saturated. I believe the community is more interested in more challenging downstream dense prediction tasks such as object detection and scene parsing.

We have conducted a primitive study on the transferability of the vanilla ViT pre-trained on mid-sized ImageNet-1k using DINO to the more challenging COCO object detection benchmark via a very simple & compact task layer called YOLOS (https://arxiv.org/abs/2106.00666). The result seems good: the AP of DINO supervised ViT-S matches the label supervised counterpart.

So the promising primitive result encourages me to ponder the question: what about the transfer learning performance of DINO on more dense prediction tasks? The specific task decoder layers well recognized by the community such as DETR (for object detection) and SETR (for semantic segmentation or scene parsing) can be almost used out of the box for the ViT family.

I wonder is there a plan on studying the transfer learning performance of DINO on downstream dense prediction tasks in the future?
